browser/ui/webui/brave_new_tab_message_handler.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019 The Brave Authors. All rights reserved.
+// Copyright (c) 2020 The Brave Authors. All rights reserved.
```

Comment:
😩👌 C'est magnifique

Reply:
Really? New pages should have 2020, but the copyright for existing pages is still the original year. Bumping it to 2020 would mean that it didn't have a copyright in 2019

Reply:
Didn't see this was an existing file - yeah, we should leave existing file notices alone 👍

Reply:
oh no

Reply:
As an aside, are we sure about that @bridiver @bsclifton, because that would imply that copyright would expire sooner than it should, considering that implementation changes later than the initial year the file was created? In other words the current algorithm in the file should be copyright for the year the algorithm was implemented? Previous versions of the file would be copyright in the previous year in the history of the repo or release.

Reply:
reverted